src/testUtils/testDefaults.ts

```diff
@@ -37,30 +39,30 @@ export const DEFAULT_REQUEST_ID = 'ARequestID'
 export const INVALID_ENTITY_ID = 'invalid/entity/id' as unknown as number
 
 export const DEFAULT_DUE_DATE = {
-    recurring: false,
+    isRecurring: false,
     string: 'a date string',
     date: DEFAULT_DATE,
 }
 
 export const INVALID_DUE_DATE = {
     ...DEFAULT_DUE_DATE,
-    recurring: 'false',
+    isRecurring: 'false',
 }
 
 export const DEFAULT_QUICK_ADD_RESPONSE: QuickAddTaskResponse = {
-    id: DEFAULT_TASK_ID,
-    projectId: DEFAULT_PROJECT_ID,
+    id: parseInt(DEFAULT_TASK_ID),
+    projectId: parseInt(DEFAULT_PROJECT_ID),
     content: DEFAULT_TASK_CONTENT,
     description: DEFAULT_TASK_DESCRIPTION,
     priority: DEFAULT_TASK_PRIORITY,
-    sectionId: DEFAULT_SECTION_ID,
-    parentId: DEFAULT_PARENT_ID,
+    sectionId: parseInt(DEFAULT_SECTION_ID),
+    parentId: parseInt(DEFAULT_PARENT_ID),
     childOrder: DEFAULT_ORDER,
-    labels: DEFAULT_LABELS,
-    responsibleUid: DEFAULT_ASSIGNEE,
+    labels: [1, 2, 3],
```

Comment:
Shouldn't these be strings?

https://developer.todoist.com/sync/v9/#quick-add-an-item